src/System.Slices/tools/ILSub/ILSub/Program.cs

```diff
@@ -52,9 +52,7 @@ static Match FindILSubClass(string s)
             string[] tokens = new string[]
             {
                 @".class private auto ansi beforefieldinit System.ILSub",
-                "}",
-                "}",
-                "\n"
+                "} // end of class System.ILSub",
```

Comment:
Tool changes look good, not sure about the IL part yet. If you checked that already it's good with me. I'll try to analyze it but it might take a while as I'm fairly newbie to IL